src/watchmaker/exceptions/__init__.py

```diff
@@ -1,18 +1,127 @@
+import datetime
 import logging
+import os
 import sys
 
 
-def SystemFatal(msg):
-    """
-    This is a temporary exception handler for the project.
+class LogHandler:
```

Comment:
It feels a little odd for this to be in exceptions. Maybe it's own module, so the import semantics are just `from watchmaker import LogHandler as log`? Also, is there some way to make the `add()` function the "default" for the class, so the usage semantics are just `log(msg, log_type, exc)`?